silkworm/silkrpc/core/evm_executor.hpp

```diff
@@ -42,12 +42,7 @@
 
 namespace silkworm::rpc {
 
-struct ExecutionResult {
-    int64_t error_code;
-    uint64_t gas_left;
-    Bytes data;
-    std::optional<std::string> pre_check_error{std::nullopt};
-};
+struct ExecutionResult;
```

Comment:
I suggest to declare `ExecutionResult` struct here moving the implementation of the `error_message` method in `evm_executor.cpp`. This will allow you to:
- avoid an unnecessary forward declaration
- hide unnecessary implementation details for `error_message` method which doesn't need to be inline

Reply:
ok